README.md

```diff
@@ -44,5 +44,5 @@ This repository contains the following packages:
 
 - Get hands-on experience with the [getting started](GETTING_STARTED.md) tutorial.
 - Check the [installation guide](INSTALLING.md) to test this on a robot.
```

Comment:
Maybe remove these two lines as the links don't work anymore.
```suggestion
```

Reply:
Removed in https://github.com/Ekumen-OS/beluga/pull/346/commits/3e9961897fcd9f2f23db5253b2d018be630b0f11.